users_app/models.py

```diff
@@ -41,11 +43,14 @@ def create_superuser(self, email, password, **extra_fields):
 class User(AbstractUser):
     username = None
     email = models.EmailField(_('email address'), unique=True)
-
     USERNAME_FIELD = 'email'
     REQUIRED_FIELDS = []
 
     objects = UserManager()
 
     def __str__(self):
         return self.email
+
+    @property
+    def is_AP(self):
+        return self.email in ALLOWED_AP_ACCOUNTS
```

Comment:
Do we consider using types of users instead of this implementation?

Reply:
Now the only way to know if a user is an AP is that his email is in a white list